### ISSUE TITLE
reloadOnPrerender option not working

### ISSUE THREAD

**Issue by archansel** [isaachinman/next-i18next#1396]
## Describe the bug
I tried to use `reloadOnPrerender` option so that my app will reload the localization when I change something in json translation files

## Occurs in next-i18next version
"next": "11.1.2",
"next-i18next": "^8.6.0",

## Steps to reproduce
- Create new nextjs app
- Replace its content with the example app from next-i18next
- Enable `reloadOnPrerender`

my `next-i18next.config.js`
```
module.exports = {
  i18n: {
    defaultLocale: 'en',
    locales: ['en', 'de'],
  },
  reloadOnPrerender: true,
}
```

## Expected behaviour
App should rerender when changing translations json files

## Screenshots
If applicable, add screenshots or a GIF to help explain your problem.

## OS (please complete the following information)
 - Device: MBP Retina 2015 13"
 - Browser: Chrome 92

## Additional context
I tried few things and none trigger rerender:
- Changing translation content
- Removing translation key

**Comment by isaachinman** [comment]
Although merged, this feature has not yet been released, as I am out of office. I will release it within a week. Thanks for your patience.

**Comment by archansel** [comment]
Ah I see, no worries

**Comment by isaachinman** [comment]
The `reloadOnPrerender` option has been released in `next-i18next@8.7.0`.
